apps/couchdb.go

```diff
@@ -78,7 +78,7 @@ func (p LoggingProcessorCouchdb) Components(tag string, uid string) []fluentbit.
 				{
 					// Format https://github.com/apache/couchdb/blob/main/src/couch_log/src/couch_log_writer_syslog.erl#L72
 					// Sample line: [notice] 2021-12-02T23:36:42.555157Z nonode@nohost <0.17165.1> a5f585a0d3 localhost:5984 127.0.0.1 otelu PUT /oteld 201 ok 16
-					Regex: `^\[(?<level>\w*)\] (?<timestamp>[\d\-\.:TZ]+) (?<node>\S+)@(?<host>[^\s]+) \<(?<pid>[^ ]*)\> [\w-]+ (?<http_request_serverIp>[^ ]*) (?<http_request_remoteIp>[^ ]*) (?<message>(?<remote_user>[^ ]*) (?<http_request_requestMethod>[^ ]*) (?<path>[^ ]*) (?<http_request_status>[^ ]*) (?<status_message>[^ ]*) (?<http_request_responseSize>[\d]*)$)`,
+					Regex: `^\[(?<level>\w*)\] (?<timestamp>[\d\-\.:TZ]+) (?<node>\S+)@(?<host>[^\s]+) \<(?<pid>[^ ]*)\> [\w-]+ (?<http_request_serverIp>[^ ]*) (?<http_request_remoteIp>[^ ]*) (?<message>(?<remote_user>[^ ]*) (?<http_request_requestMethod>[^ ]*) (?<http_request_requestUrl>[^ ]*) (?<http_request_status>[^ ]*) (?<status_message>[^ ]*) (?<http_request_responseSize>[\d]*)$)`,
```

Comment:
There might be a bug with the current CouchDB implementation. The `path` is not prefixed with `http_request_`, as a result it might not be populated as LogEntry.httpRequest.RequestUrl in theory.

Still running integration tests.

Reply:
Actually it's clearly documented as "jsonPayload.path": https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/couchdb#monitored-logs

Might be too late to change at this point. Perhaps next major release.